PWGCF/DataModel/FemtoDerived.h

```diff
@@ -305,31 +312,39 @@ DECLARE_SOA_TABLE(FDHfCand, "AOD", "FDHFCAND", //! Table to store the derived da
                   fdhf::Phi<fdhf::Prong0Pt, fdhf::Prong0Phi, fdhf::Prong0Eta, fdhf::Prong1Pt, fdhf::Prong1Phi, fdhf::Prong1Eta, fdhf::Prong2Pt, fdhf::Prong2Phi, fdhf::Prong2Eta>,
                   fdhf::Pt<fdhf::Prong0Pt, fdhf::Prong0Phi, fdhf::Prong0Eta, fdhf::Prong1Pt, fdhf::Prong1Phi, fdhf::Prong1Eta, fdhf::Prong2Pt, fdhf::Prong2Phi, fdhf::Prong2Eta>);
 
-DECLARE_SOA_TABLE(FDResultsHF, "AOD", "FDRESULTSHF", //! table to store results for HF femtoscopy
+DECLARE_SOA_TABLE(FDResultsCharm, "AOD", "FDRESULTSCHARM", //! table to store results for HF femtoscopy
+                  fdhf::GIndexCol,
+                  fdhf::TimeStamp,
                   fdhf::CharmM,
                   fdhf::CharmPt,
-                  fdhf::PtAssoc,
+                  fdhf::CharmEta,
+                  fdhf::CharmPhi,
+                  fdhf::Charge,
                   fdhf::BDTBkg,
                   fdhf::BDTPrompt,
                   fdhf::BDTFD,
-                  fdhf::Kstar,
-                  fdhf::KT,
-                  fdhf::MT,
-                  fdhf::Mult,
-                  fdhf::MultPercentile,
-                  fdhf::Charge,
-                  fdhf::PairSign,
-                  fdhf::ProcessType,
                   fdhf::FlagMc,
                   fdhf::OriginMcRec);
 
-DECLARE_SOA_TABLE(FDResultsHFTrkInfo, "AOD", "FDRESULTSHFTRKINFO", //! table to store results for HF femtoscopy
+DECLARE_SOA_TABLE(FDResultsTrk, "AOD", "FDRESULTSTRK", //! table to store results for HF femtoscopy
```

Comment:
Please, keep `HF` as before in the table declaration

Reply:
Ok, Fixed